platforms/azure/package.json

```diff
@@ -36,5 +36,5 @@
     "ssh2": "^1.4.0",
     "json-schema": "^0.4.0"
   },
-  "version": "4.15.1"
+  "version": "4.16.0"
```

Comment:
Actually this will no longer be needed if we decide we want to go with: https://github.com/Unleash/unleash-docker-community/pull/28